refinery/data_set_manager/utils.py

```diff
@@ -9,22 +9,19 @@
 import time
 import urlparse
 
+import core
```

Comment:
We won't be able to automate everything and it's OK because we do have a coding style guide where [Python module import order](https://github.com/refinery-platform/refinery-platform/wiki/coding-style-guide#python) is described. It is also a part of [PEP8](https://www.python.org/dev/peps/pep-0008/#imports).